src/ncp/ncp_buffer.hpp

```diff
@@ -381,36 +382,36 @@ class NcpFrameBuffer
     otError         OutFramePrepareMessage(void);
     otError         OutFrameFillMessageBuffer(void);
 
-    uint8_t * const  mBuffer;                    // Pointer to the buffer used to store the data.
-    uint8_t * const  mBufferEnd;                 // Points to after the end of buffer.
-    const uint16_t   mBufferLength;              // Length of the the buffer.
+    uint8_t * const mBuffer;                    // Pointer to the buffer used to store the data.
```

Comment:
Thats odd, if I run ``make pretty`` on a clean version of master, I dont see it trying to change this file.  I think these lines should be reverted.  Nothing actually changed on these lines, and the formatting didn't even change all the lines, so its created some skew in the indentations.
We are slowly switching from astyle to uncrustify anyways.